### ISSUE TITLE
Question about training batch size of pre-trained model.

### ISSUE THREAD

**Issue by TheCuriousJoe** [metric-anything/metric-anything#2]
I’m not sure I understand this part. You mention that the training dataset contains 20 million samples. 
<img width="1966" height="126" alt="Image" src="https://github.com/user-attachments/assets/5a3f9d20-b377-43fb-bf03-8193e0c1e642" />
What batch size did you use during training? If the batch size is 1, then 100k steps on 144 GPUs would process about 14.4 million samples, which is still less than one epoch. Is my understanding correct?
<img width="1976" height="180" alt="Image" src="https://github.com/user-attachments/assets/02f061af-60b3-4a64-8138-e5faccdd3472" />
